node/resolvers/subscriptions/index.ts

```diff
@@ -0,0 +1,22 @@
+export const queries = {
+  subscriptionsCountByStatus: async (_, args, { dataSources: { subscription } }) => {
+    const options = {
+      where: `createdAt between ${args.initialDate} and ${args.endDate}`,
+      schema: "bi-v1",
+      field: "status",
+      type: "count",
+      interval: "day"
+    }
+
+    return await subscription.subscriptionAggregations(options).then((data) => {
```

Comment:
No need to `await` followed by `then` :) pick one!

Reply:
👍